apache-rat-core/src/main/java/org/apache/rat/config/AddLicenseHeaders.java

```diff
@@ -23,25 +23,10 @@
  * Value of addLicenseHeaders configuration option.
  */
 public enum AddLicenseHeaders {
-    /** Add license headers to a *.new file*/
-    TRUE, 
-    /** Do not add headers */
-    FALSE, 
-    /** Add headers and overwrite the old file */
-    FORCED;
-
-    private static final char SEPARATOR = '|';
-
-    /**
-     * @return pretty-printed list of all available options.
-     */
-    public static String getValuesForHelp() {
```

Comment:
@Claudenw does a call to this method miss somewhere in the utils module in order to print all available methods?